### PR TITLE
k8s-qdrant bundle

### PR DESCRIPTION
**Pull request by likelymichael**
Bundle for k8s-qdrant deploying to k8s cluster. Need to create artifact definition next

### CODE REVIEW

File: massdriver.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+access: public
+type: infrastructure
+
+params:
+  required:
+    - namespace
+    - instance_configuration
+    - replica_configuration
+  properties:
+    namespace:
```
Comment (chrisghill) [inline massdriver.yaml:13]: I would suggest nesting these under a top level block, like `database` or something.

File: massdriver.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          title: Memory Limit
+          description: Unit is Gi. Decimal numbers are allowed. Value must be between 0.5 and 64
+          minimum: 0.5
+          maximum: 64
+    replica_configuration:
+      type: object
+      title: Replica Configuration
+      properties:
+        replicas:
+          type: integer
```
Comment (chrisghill) [inline massdriver.yaml]: Might consider turning this into an enum if there are only 5 values

File: massdriver.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          title: Replicas
+          description: Number of replicas to create. Must be an integer between 1 and 5.
+          minimum: 1
+          maximum: 5
+
```
Comment (chrisghill) [inline massdriver.yaml:67]: Probably want to include storage configuration

File: src/helm_values.tf [hunk 4]
```diff
@@ -0,0 +1,18 @@
+locals {
+  helm_values = {
+    nameOverride = var.md_metadata.name_prefix
+    podLabels    = var.md_metadata.default_tags
+    replicaCount = var.replica_configuration.replicas
+    resources = {
+      requests = {
```
Comment (chrisghill) [inline src/helm_values.tf:7]: Usually requests is less than limits - like 1/2 or 2/3 of the limit. You can make them the same, but it effectively guarantees that you'll either be CPU throttled, have OOMKilled pods, or be over-provisioned.

File: src/main.tf [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  namespace        = var.namespace
+  create_namespace = true
+
+  values = [
+    "${file("${path.module}/values.yaml")}",
+    yamlencode(local.helm_values)
+  ]
+
+  depends_on = [random_password.api-key]
+}
```
Comment (chrisghill) [inline src/main.tf:26]: Probably should include k8s monitoring stuff.

**Review by chrisghill — COMMENTED** [review]
Some suggested changes.

File: massdriver.yaml (hunk 1 above)
Comment (likelymichael) [reply massdriver.yaml:13]: Done

File: src/main.tf (hunk 5 above)
Comment (likelymichael) [reply src/main.tf:26]: Added

**Review by chrisghill — APPROVED** [review]

**Pull request merged**